src/components/PresentationContext.tsx

```diff
@@ -2,6 +2,8 @@ import { createContext, PropsWithChildren, useContext, useMemo } from "react";
 import { GridStyle } from "src/components/Table";
 import { Typography } from "src/Css";
 
+export type InputStylePalette = "success" | "warning" | "caution" | "info";
```

Comment:
Mirrors our `coloredChips` and does leave room for more bespoke customization in the future if the need arises